HowTo build program Hello PLCnext/readme.md

```diff
@@ -1,52 +1,55 @@
-# How to build a simple "Hello PLCnext" application #
+# How to build a simple "Hello PLCnext" application
 
 > Based on the official [documentation](https://github.com/PLCnext/SampleRuntime/blob/master/getting-started/Part-01/README.md).
 
-## Using command line (pure **CMake**) ##
+## Using command line (pure **CMake**)
 
 After cloning repository execute this steps inside `"HowTo build program Hello PLCnext\Hello-PLCnext"` working direcrory.
 
 Configure:
+
 ```sh
 cmake --preset=build-windows-AXCF2152-2021.0.3.35554 .
 ```
 
 Build:
+
 ```sh
 cmake --build --preset=build-windows-AXCF2152-2021.0.3.35554 --target all
 ```
 
 Deploy:
+
 ```sh
 cmake --build --preset=build-windows-AXCF2152-2021.0.3.35554 --target install
 ```
 
 In above commands the `build-windows-AXCF2152-2021.0.3.35554` preset name specifies build parameters on OS Windows for AXC 2152. It is stored in `"HowTo build program Hello PLCnext\Hello-PLCnext\CMakePresets.json"` file. Inside this file you can found parameters:
 
- - The `PLCNEXT_SDK_ROOT` parameter specifies the full path to the root directory of the SDK.
- - The `ARP_DEVICE` and `ARP_DEVICE_VERSION` parameters should specify the SDK device and version.
+- The `PLCNEXT_SDK_ROOT` parameter specifies the full path to the root directory of the SDK.
+- The `ARP_DEVICE` and `ARP_DEVICE_VERSION` parameters should specify the SDK device and version.
 
 You can change it, if necessary, to the SDK device, version and path of the SDK that you are using.
 
 After you can find executable here:
->deploy\AXCF2152_21.0.3.35554\Release\bin\hello_PLCnext
+
+> deploy\AXCF2152_21.0.3.35554\Release\bin\hello_PLCnext
 
 On **Linux OS** use `build-linux-AXCF2152-2021.0.3.35554` preset name.
 
-## Using MS Visual Code ##
+## Using MS Visual Code
 
 Open directory `"HowTo build program Hello PLCnext\Hello-PLCnext"` in MS Visual Code.
+To work correctly, you need to install the CMake, CMake Tools extension. You also need to specify the path to ninja.exe to build the project. Open the CMakeCache file, which has the path How To build programm `"Hello PLCnext\Hello-PLCnext\bin\AXCF2152_2020.6.1"` In this file, look for the line `"CMAKE_MAKE_PROGRAM:FILEPATH="`and write here the path to ninja.exe `"C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\Ninja\ninja.exe"` Save the project, restart, and build the project.
```

Comment:
It's not a good idea to change automatically generated files. Better is modifying the PATH environment variable so that Windows knows where to find the Ninja executable, depending on your setup (but it should be done during setup). 
```suggestion
To work correctly, you need to install the CMake, CMake Tools extension, ninja.
```